client/views/room/contextualBar/Apps/Apps.tsx

```diff
@@ -0,0 +1,31 @@
+import { ButtonGroup, Button, Box } from '@rocket.chat/fuselage';
+import { UiKitComponent, UiKitModal, modalParser } from '@rocket.chat/fuselage-ui-kit';
+import React from 'react';
+
+import VerticalBar from '../../../../components/VerticalBar';
+
+const Apps = ({ view, onSubmit, onClose }): JSX.Element => (
+	<>
+		<VerticalBar.Header>
+			{view.showIcon ? <VerticalBar.Icon name='keyboard' /> : null}
+			<VerticalBar.Text>{'App bar'}</VerticalBar.Text>
```

Comment:
This title should come from the `view`

Reply:
It is comming from an object with the app's name and id